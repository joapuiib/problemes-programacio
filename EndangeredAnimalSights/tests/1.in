3
Mustela putorius
fura de bosc
Canis lupus
Llop
Testudo hermanni
Tortuga mediterr√†nia
5
0 2
0 1
1 1
2 1
0 2

