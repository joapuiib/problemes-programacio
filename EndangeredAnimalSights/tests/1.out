--- Avistaments ---
Mustela putorius - fura de bosc: 2
Mustela putorius - fura de bosc: 1
Canis lupus - Llop: 1
Testudo hermanni - Tortuga mediterrània: 1
Mustela putorius - fura de bosc: 2
--- Resum ---
Mustela putorius - fura de bosc: 5
Canis lupus - Llop: 1
Testudo hermanni - Tortuga mediterrània: 1
